### ISSUE TITLE
Script can reach max_execution_time during image thumbnail creation

### ISSUE THREAD

**Issue by ghost** [trippo/ResponsiveFilemanager#126]
When I tested RFM with a lot of images, I copied a large number of files in the same time into the source folder. Then RFM started generating the thumbnails, which didn't finish in 30 seconds. That's why a "Fatal error: Maximum execution time of 30 seconds exceeded" was thrown.

My workaround for this is very easy but a bit nasty and I doesn't fit for the most shared web hosted sites. So there is a `create_img_gd` function in the utils.php file. I modified it in the following way:

``` php
function create_img_gd($imgfile, $imgthumb, $newwidth, $newheight="",$option="crop") {
    $timeLimit= ini_get('max_execution_time');
    set_time_limit(30);
    $result= false;
    if(image_check_memory_usage($imgfile,$newwidth,$newheight)){
    require_once('php_image_magician.php');
    $magicianObj = new imageLib($imgfile);
    $magicianObj -> resizeImage($newwidth, $newheight, $option);
    $magicianObj -> saveImage($imgthumb,80);
    $result= true;
    }
    set_time_limit($timeLimit);
    return $result;
}
```

Basically the first `set_time_limit` function sets the PHP time counter to 0 (and the limit to 30), and finally the original time limit is restored by the second invocation of it.

Decide upon adopting it. Personally, I don't know if it creates security issues or not.
